### DISCUSSION TITLE
The mail app permit me receive messages from people in my address list

### DISCUSSION THREAD

**Discussion by lumenpink** [beakerbrowser/beaker#1555] [General] [answered]
There is no way to "ping" a user without him add me first?

**Comment by pfrazee** [comment, marked as answer]
Yeah there's not currently any way to ask people to add you to their contacts list. We need to find a good ole' decentralized, long-term infrastructure solution.

There *may* be a solution possible via PeerSockets. It might also be something that could be solved via a service, which we'd want to create an open standard for.

**Reply by lumenpink** [reply]
That's good news. Hope to see this soon! Anyway Beaker still Rulez!

**Comment by danrobi11** [comment]
hello. im just curious. whats the mail app?

**Reply by pfrazee** [reply]
It's a little POC I put together hyper://1bc1faf01a22270fb5698a60e63ef7a596ad976457e6d9914a8fd56d87281917/
